package.json

```diff
@@ -22,6 +22,34 @@
     "sys/",
     "testing/"
   ],
+  "exports": {
+    ".": {
+      "import": "./internal/stencil-core/index.js",
+      "require": "./internal/stencil-core/index.cjs"
+    },
+    "./internal": {
+      "import": "./internal/index.js"
+    },
+    "./internal/client": {
+      "import": "./internal/client/index.js"
+    },
+    "./internal/testing": {
+      "import": "./internal/testing/index.js"
+    },
+    "./internal/testing/*": {
+      "import": "./internal/testing/*"
+    },
+    "./internal/app-data": {
+      "import": "./internal/app-data/index.js",
+      "require": "./internal/app-data/index.cjs"
```

Comment:
Still think it's so weird we expose things under "internal" 😅

Reply:
Totally! These aren't user interfaces though , they are all just import references that Stencil is using.

Reply:
you could remove them, but then your imports should use relative paths, with js extension

Reply:
Yep, we will do more optimizations in this direction, e.g. move to ESM